capabilities/translation/analyze-plain-text-sentiment/maps/google-apis-language.suma

```diff
@@ -0,0 +1,72 @@
+profile = "translation/analyze-plain-text-sentiment@1.0"
+provider = "google-apis-language"
+
+map AnalyzePlainTextSentiment {
+
+  http POST "/v1/documents:analyzeSentiment" {
+    security "apikey"
+    request {
+      body {
+        encodingType = 'UTF8',
+        document = {
+          type: 'PLAIN_TEXT',
+          language: input.languageCode,
+          content: input.text
+        }
+      }
+    }
+
+    response 200 "application/json" {
+      map result {
+        sentiment = call MapScoreToSentiment(score = body.documentSentiment.score)
+        score = body.documentSentiment.score
+      }
+    }
+
+    response 400 "application/json" {
+      map error {
+        title = (body.error.status === "INVALID_ARGUMENT") ? "Invalid argument" : "Bad request"
+        detail = body.error.message
+      }
+    }
+
+    response 401 "application/json" {
+      map error {
+        title = "Unauthorized"
+        detail = body.error.message
+      }
+    }
+
+    response 403 "application/json" {
+      map error {
+        title = "Forbidden"
+        detail = body.error.message
+      }
+    }
+
+    response 500 "application/json" {
+      map error {
+        title = "Internal server Error"
+        detail = body.error.message
+      }
+    }
+  }
+}
+
+operation MapScoreToSentiment {
+  set {
+    result = (() => {
```

Comment:
This is OK but I wonder if we do not have a nicer way to do this common switch / mutliple ifs task e.g. using `set if`

Reply:
@TheEdward162  what is the nicest way to do this? :D